Documentation/SpiderBasic/English/TouchScreen.txt

```diff
@@ -51,7 +51,7 @@
 @ReturnValue
   The finger 'x' movement (in pixels) since the last call of this function. 
   It can be either a negative or positive value, depending on whether or not the finger has been moved to 
-  left or right since the last call. @@ExamineTouchScreen should be called before this function 
+  the left or right since the last call. @@ExamineTouchScreen should be called before this function 
```

Comment:
I'm not english native speaker so I may be wrong and stick to French grammar rules. For me that feel weird to have `the` before `left` and not before `right`.